### ISSUE TITLE
Crosscompiling for Mac - File not found for module bsd

### ISSUE THREAD

**Issue by mxinden** [dvc94ch/if-watch#1]
Crosscompiling this crate on Linux for Mac fails with:

```bash
cargo build --target x86_64-apple-darwin

error[E0583]: file not found for module `bsd`
  --> src/unix.rs:38:1
   |
38 | mod bsd;
   | ^^^^^^^^
   |
   = help: to create the module `bsd`, create file "src/unix/bsd.rs"

error[E0433]: failed to resolve: use of undeclared crate or module `linux`
  --> src/unix.rs:35:16
   |
35 | type Watcher = linux::NetlinkSocket;
   |                ^^^^^ use of undeclared crate or module `linux`
```

In case there is no plan to support Mac in this crate, I would suggest removing the `mod bsd` declaration and extending the documentation by the supported platforms.

**Comment by dvc94ch** [comment]
It's removed in master. I'd love for someone to contribute mac/ios support, but I don't have the devices and the company I work for doesn't care about the platform as it isn't something the customers use.

**Comment by dvc94ch** [comment]
Closed in #2
